care/emr/api/viewsets/tag_config.py

```diff
@@ -55,6 +55,24 @@ class TagConfigViewSet(
     filter_backends = [filters.DjangoFilterBackend, OrderingFilter]
     ordering_fields = ["priority", "created_date", "modified_date"]
 
+    def validate_data(self, instance, model_obj=None):
+        filters = {"slug": instance.slug}
+        if model_obj:
+            if model_obj.organization:
+                filters["organization"] = model_obj.organization
```

Comment:
The organization is not a valid filter here, slugs should be unique across all orgs in a. facility